src/test/scala/de/zalando/beard/parser/BeardTemplateParserSpec.scala

```diff
@@ -351,49 +349,86 @@ class BeardTemplateParserSpec extends FunSpec with Matchers {
 
   describe("from file") {
     it("should parse the template") {
-      val template = Source.fromInputStream(getClass.getResourceAsStream("/templates/hello.beard")).mkString
+      val template = Source.fromInputStream(
+        getClass.getResourceAsStream("/templates/hello.beard")
+      ).mkString
 
       BeardTemplateParser(template).statements should be(
-        Seq(
+        List(
           Text("<div>somediv</div>"),
-          BlockStatement(Identifier("navigation"), List(NewLine(1),
-            White(4),
-            Text("<ul>"),
-            NewLine(1),
-            White(8),
-            Text("<li>first</li>"),
-            NewLine(1),
-            White(4),
-            Text("</ul>"))),
-          NewLine(2),
-          Text("<p>Hello"),
-          White(1),
-          Text("world</p>"),
-          NewLine(2),
+          Text("\n"),
+          BlockStatement(
+            Identifier("navigation"),
+            List(
+              Text("    <ul>"),
+              Text("\n"),
+              Text("        <li>first</li>"),
+              Text("\n"),
+              Text("    </ul>"),
+              Text("\n"
+              )
+            )
+          ),
+          Text("\n"),
+          Text("<p>Hello world</p>"),
+          Text("\n"),
+          Text("\n"),
           IfStatement(
-            CompoundIdentifier("usersExist"),
-            List(NewLine(1),
-            White(4),
-            Text("<div>No"),
-            White(1),
-            Text("users</div>"),
-            NewLine(1)), List(NewLine(1),
-            White(4),
-            Text("<div"),
-            White(1),
-            Text("class=\"users\">"),
-            ForStatement(Identifier("user"), CompoundIdentifier("users", List()), List(NewLine(1),
-              White(8),
-              IdInterpolation(CompoundIdentifier("user", List("name"))),
-              IfStatement(
-              CompoundIdentifier("user", Seq("isLast")),
-                List(Text(",")), List()), White(8),
-              RenderStatement("user-details", List(AttributeWithIdentifier("user", CompoundIdentifier("user", List())),
-                AttributeWithValue("class", "default"))))),
-            NewLine(1),
-            White(4),
-            Text("</div>"),
-            NewLine(1)))
+            CompoundIdentifier(
+              "usersExist",
+              List()
+            ),
+            List(
+              Text("    <div>No users</div>"),
+              Text("\n")
+            ),
+            List(
+              Text("    <div class=\"users\">"),
+              Text("\n"),
+              Text("    "),
```

Comment:
Do you think these whitespaces before `for` statements should be removed? I thought about it, and if I was using this tool to generate some python code, for example, I would not want beard to remove any white spaces since they can mean something.